action.yml

```diff
@@ -18,7 +18,7 @@ runs:
         bootstrap_version='v1.0.0'
         expected_bootstrap_version_digest='e36a05ab402bfee5463ad4752d8dc2941204c7b01a9a9931f921e91d94ba2484'
         curl -L https://storage.googleapis.com/cosign-releases/v1.0.0/cosign-linux-amd64 -o cosign
```

Comment:
unrelated, but we might want to use here `https://storage.googleapis.com/cosign-releases/${bootstrap_version}/cosign-linux-amd64` instead. wdyt ?

Reply:
nice catch 👏